packages/generate-component-docs/src/__snapshots__/contract.test.ts.snap

```diff
@@ -8207,12 +8209,12 @@ exports[`filterSuggestions 1`] = `
   props: {
```

Comment:
Looks like this export is incorrectly treated as a component, hence the bogus definition for "props".